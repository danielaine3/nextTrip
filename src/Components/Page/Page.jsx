import React from 'react'
import { Container, Typography, makeStyles } from '@material-ui/core'

const useStyles = makeStyles(() => ({
  title: {
    padding: 10,
  },
}))

export const Page = ({children, title}) => {
  const classes =  useStyles()
  return (
      <Container maxWidth='sm'>
        <Typography variant='h5' className={classes.title}>
          {title}
        </Typography>
        {children}
      </Container>
  )
}
