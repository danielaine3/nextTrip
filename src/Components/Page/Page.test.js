import React from 'react'
import { render} from '@testing-library/react'
import { Page } from './Page'

describe('<Page />', () => {
  it('should render Page', async () => {
    const { asFragment } = await render(
    <Page title={'title'}>
      <div>Test Child</div>
      <div>Test Child</div>
    </Page>
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
