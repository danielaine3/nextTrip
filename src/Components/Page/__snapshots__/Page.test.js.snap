// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Page /> should render Page 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root MuiContainer-maxWidthSm"
  >
    <h5
      class="MuiTypography-root makeStyles-title-1 MuiTypography-h5"
    >
      title
    </h5>
    <div>
      Test Child
    </div>
    <div>
      Test Child
    </div>
  </div>
</DocumentFragment>
`;
